### ISSUE TITLE
[Feature request] card working with attribues

### ISSUE THREAD

**Issue by Tom-ahawk** [aukedejong/lovelace-windrose-card#28]
Please consider making this nice card work with attributes and not only entities like:
```
  wind_direction_entity: 
    - entity: weather.home
    - attribute: wind_bearing
```
or 
`  entity: '{{ state_attr("weather.home", "wind_bearing") }}' `

Creating new sensors from entities, for making this work, creates just more sensors to maintain....

**Comment by aukedejong** [comment]
Good idea. I thought about it, but decided to wait until someone needs it.
Maybe in the next feature version. I'll keep you updated in this issue.

**Comment by Egainmoney** [comment]
hello, I am setting it up on my side and I have the same problem and this solution interests me. I also have the direction of the wind which is in attributes of an entity. thanks for the work

**Comment by elupus** [comment]
Indeed. Would also be nice to just work directly from a standard weather entity: https://www.home-assistant.io/integrations/weather/ since the attributes are standardized there.

**Comment by aukedejong** [comment]
Currently working on the original feature request.
I'm not making it work directly with a standard weather entity yet. You still need to set the attribute values.

I think that should be an other feature request, separate from this one. Can you create it, @elupus ?
It also needs some more thinking.
For example, should the card show the normal wind speed and gust speed both automatically, as 2 bars?
How will this fit in the current yaml config structure.

**Comment by aukedejong** [comment]
This is introduced in version 1.14.0
Upcoming release 1.15.0 will also support auto speed unit determination for weather entity's.
